web/crux/src/app/deploy/interceptors/deploy.start.interceptor.ts

```diff
@@ -91,6 +91,30 @@ export default class DeployStartValidationInterceptor implements NestInterceptor
       })
     }
 
+    if (!deployment.protected) {
+      const {
+        query: { ignoreProtected },
+      } = req
+
+      if (!ignoreProtected || ignoreProtected === false) {
```

Comment:
Is there a case when ignoreProtected is `!ignoreProtected` is `true` but it equals `false` ?

Reply:
I wanted to handle the case when `ignoreProtected` is null, but yeah they are the same. I'll remove the second condition.